testsuite/scripts/samples/test_grand_canonical.py

```diff
@@ -20,7 +20,7 @@
 
 sample, skipIfMissingFeatures = importlib_wrapper.configure_and_import(
     "@SAMPLES_DIR@/grand_canonical.py", n_int_cycles=51, n_int_steps=5,
-    warm_n_times=10, warm_steps=50, cmd_arguments=[0.01, 0.01])
+    warm_n_times=10, warm_steps=50, cmd_arguments=[8.523659461370000200e-04, -3.365753246398797693e-01])
```

Comment:
how many significant figures do we really need here?

Reply:
8.523659461370000200e-04 was the concentration I used for determining the excess chemical potential. This is therefore exact. 
The value for the excess chemical potential has a standard error of  +/- 1.590969027075880404e-03 therefore I cropped the result to the third digit after the point: -0.336

Reply:
you're providing a concentration with 19 significant figures... either WLRE relies on chaos theory, or you're not familiar with Avogadro's constant :) please trim that down, espresso runs on multiple CPU architectures with different machine precision, your value doesn't even fit inside a `long double` on a 64bit machine